examples/example.js

```diff
@@ -15,7 +15,8 @@ const schema = {
 
 fastify
   .get('/', schema, function (req, reply) {
-    reply(null, { hello: 'world' })
+    reply.header('Content-Type', 'application-json').code(200)
+    reply.reply(null, { hello: 'world' })
```

Comment:
what are your thoughts for error handling?

Also, while we are touching the handler signature, we should also think into supporting promises (sigh).